services/sidecar/Dockerfile

```diff
@@ -95,21 +104,17 @@ RUN $SC_PIP install -r requirements/prod.txt &&\
 #  + /home/scu     $HOME = WORKDIR
 #    + services/sidecar [scu:scu]
 #
-FROM cache as production
+FROM base as production
 
 
-ENV SC_BUILD_TARGET production
+ENV SC_BUILD_TARGET=production
 ENV PYTHONOPTIMIZE=TRUE
 
 WORKDIR /home/scu
 
-RUN mkdir -p services/sidecar &&\
-    chown scu:scu services/sidecar &&\
-    mv /build/services/sidecar/docker services/sidecar/docker &&\
-    rm -rf /build
-
-RUN apk del --no-cache\
-      gcc
+COPY --from=cache ${VIRTUAL_ENV} ${VIRTUAL_ENV}
```

Comment:
Can these two lines be collapsed into
`` COPY --chown=scu:scu --from=cache  ${VIRTUAL_ENV} ${VIRTUAL_ENV}``??